fidesctl/src/fidesapi/main.py

```diff
@@ -70,7 +71,13 @@ async def log_request(request: Request, call_next: Callable) -> Response:
 @app.get("/health", tags=["Health"])
 async def health() -> Dict:
     "Confirm that the API is running and healthy."
-    return {"data": {"message": "Fidesctl API service is healthy!"}}
+    server_version = str(fidesctl.__version__)
+    return {
+        "data": {
+            "message": "Fidesctl API service is healthy!",
+            "version": server_version,
+        }
+    }
```

Comment:
Since we're touching this code, it might be a good opportunity to improve the auto-generated docs using [the `responses=` argument](https://fastapi.tiangolo.com/advanced/additional-responses/?h=responses#additional-response-with-model).

Reply:
what did you have in mind here? That's for additional responses, to get a model for the current return we'd need to write a model for it and put it as the `response_model` I think

Reply:
feel free to commit so we can avoid back and forth if you know what you're looking for